var/spack/repos/builtin/packages/py-h5py/package.py

```diff
@@ -42,34 +43,34 @@ class PyH5py(PythonPackage):
     depends_on("python@3.7:", type=("build", "run"), when="@3.2:")
 
     # Build dependencies
-    depends_on("py-cython@0.23:", type="build", when="@:2")
-    depends_on("py-cython@0.29:", type=("build"), when="@3: ^python@:3.7")
-    depends_on("py-cython@0.29.14:", type=("build"), when="@3: ^python@3.8.0:3.8")
-    depends_on("py-cython@0.29.15:", type=("build"), when="@3: ^python@3.9.0:")
+    depends_on("py-cython@0.23:0", type="build", when="@:2")
+    depends_on("py-cython@0.29:0", type=("build"), when="@3:")
+    depends_on("py-cython@0.29.14:0", type=("build"), when="@3:3.7 ^python@3.8.0:3.8")
+    depends_on("py-cython@0.29.15:0", type=("build"), when="@3:3.7 ^python@3.9.0:")
     depends_on("py-pkgconfig", type="build")
     depends_on("py-setuptools", type="build")
+    depends_on("py-setuptools@61:", type="build", when="@3.8.0:")
     depends_on("py-wheel", type="build", when="@3:")
 
     # Build and runtime dependencies
     depends_on("py-cached-property@1.5:", type=("build", "run"), when="^python@:3.7")
```

Comment:
Is this still required?

Reply:
Looks like it was actually removed in 3.7.0 (see https://github.com/h5py/h5py/commit/d3980d3931eaa6e75066334b3d293f679d34037a), fixed.